src/praisonai-agents/praisonaiagents/tools/shell_tools.py

```
@@ -151,6 +153,74 @@ def execute_command(
+
+        def _emit(line, stream_name):
+            if _sink is None:
+                return
+            try:
+                _sink(_stream_events.StreamEvent(
+                    type=_stream_events.StreamEventType.TOOL_PROGRESS,
+                    content=line,
+                    metadata={"stream": stream_name},
+                ))
+            except Exception:
+                pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)